### DISCUSSION TITLE
Do dotnet/runtime patches find their way into dotnet/framework/sdk images?

### DISCUSSION THREAD

**Discussion by rufinio** [microsoft/dotnet-framework-docker#910] [Q&A]
Do dotnet/runtime patches find their way into dotnet/framework/sdk images?

If yes, how can I explore which dotnet/runtime patches are included in dotnet/framework/sdk images?

In my specific case, I'm interested in whether the patch https://github.com/dotnet/runtime/pull/52690 is available in dotnet/framework/sdk:4.8-windowsservercore-ltsc2022.

**Comment by mthalman** [comment]
As a general rule, no they are not always ported. In the specific case of https://github.com/dotnet/runtime/pull/52690, it was important to improve the functionality of .NET Framework 4.8 in container environments so those changes were ported and announced in https://github.com/microsoft/dotnet-framework-docker/issues/849. So yes, that functionality is available today for that image tag.
